hystrix-core/src/main/java/com/netflix/hystrix/strategy/properties/archaius/HystrixDynamicPropertiesArchaius.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.hystrix.strategy.properties.archaius;
+
+import com.netflix.config.PropertyWrapper;
+import com.netflix.hystrix.strategy.properties.HystrixDynamicProperties;
+import com.netflix.hystrix.strategy.properties.HystrixDynamicProperty;
+
+/**
+ * This class should not be imported from any class in core or else Archaius will be loaded.
+ * @author agentgt
+ * @see HystrixArchaiusHelper
```

Comment:
This doesn't resolve.  Can you add the {@link XXX} syntax and either import `HystrixArchaiusHelper` or use the fully-qualified class name?